Archer v2 - edited adding an api endpoint (#14486)
demisto/content: Packs/ArcherRSA/ReleaseNotes/1_1_22.md

#### Integrations
##### RSA Archer v2
- Updated the Docker image to: *demisto/python3:3.9.6.24019*.

> - fixed an issue where the *api_endpoint* parameter was not added properly to the URL.

Reviewer: missed it the last time...
```suggestion
- Fixed an issue where the *api_endpoint* parameter was not added properly to the URL.
```

Reviewer 2: Use the display name `Advanced: API Endpoint`
customers can't see(and do not mind) the `name` key.

> - fixed an issue where the *api_endpoint* paramter was not added properly for the url , after releasing the previous new version.

Reviewer: ```suggestion
- fixed an issue where the *api_endpoint* parameter was not added properly to the URL.
```
